src/AdminConsole/Pages/Organization/Settings.cshtml.cs

```diff
@@ -70,5 +76,32 @@ public async Task<IActionResult> OnPostDeleteAsync()
         return RedirectToPage();
     }
 
-    public bool CanDelete => !Organization.Applications.Any();
+    /// <summary>
+    /// Organization Id
+    /// </summary>
+    [BindProperty(SupportsGet = true)]
+    public int Id { get; set; }
+
+    /// <summary>
+    /// Organization Name
+    /// </summary>
+    [BindProperty(SupportsGet = true)]
+    public string Name { get; set; }
+
+    /// <summary>
+    /// The organization's name which is confirmed by the end user to allow the organization to be deleted.
+    /// </summary>
+    [BindProperty(SupportsGet = true)]
+    public string NameConfirmation { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Whether the organization can be deleted.
+    /// </summary>
+    public bool CanDelete => ApplicationsCount == 0;
+
+    /// <summary>
+    /// The amount of active applications belonging to the organization.
+    /// </summary>
+    [BindProperty(SupportsGet = true)]
+    public int ApplicationsCount { get; set; }

```

Comment:
@jonashendrickx Is there a reason why we allow these parameters to be bound? This allows someone to setup through querystrings.

Reply:
Just a leftover mistake.